ReactNativeClient/lib/joplin-renderer/utils.js

```diff
@@ -138,7 +138,7 @@ utils.imageReplacement = function(ResourceModel, src, resources, resourceBaseUrl
 	}
 
 	const mime = resource.mime ? resource.mime.toLowerCase() : '';
-	if (ResourceModel.isSupportedImageMimeType(mime)) {
+	if (ResourceModel.isSupportedMimeType(mime) != 'unknown') {
 		let newSrc = `./${ResourceModel.filename(resource)}`;
 		if (resourceBaseUrl) newSrc = resourceBaseUrl + newSrc;
 		return {
```

Comment:
If you add the result to from isSupportedMimeType to this return statement, then you'll be able to use it in the rules. This won't make your code any more functional, but it will lay the foundation for future changes of this type.

Reply:
With isSupportedMimeType returning the type, we can basically add one line to the audio.js file to also support video files. Should this be implemented here (change the name to audiovideo.js) or have both an audio.js and video.js?